web/web/static/web/js/components/alerts.js

```diff
@@ -0,0 +1,38 @@
+function AlertsController($scope, $filter, $log) {
+    var self = this;
+
+    self.$onInit = function() {
+        self.alerts = {
```

Comment:
I think I would prefer alerts to be listed in order that they are received regardless of the type of alert. Can we make the component maintain just a list of alerts and move the type of alert into the alert object?